src/firebase/authService.js

```diff
@@ -0,0 +1,48 @@
+import firebase from '@firebase/app';
+import '@firebase/auth';
+import userService from './userService';
+
+const authService = function() {
+    let auth = null;
+
+    function init() {
```

Comment:
What's the difference between init() and constructor()?

Reply:
I use init for actual logic to initialize the component.

I reserve the construct for initializing state.

Somewhere on the interwebs there is an explanation. I'm on mobile now, so I can link to it later